librz/main/rizin.c

```diff
@@ -184,18 +184,45 @@ static int main_help(int line) {
-			" RZ_USER_PLUGINS %s\n"
-			" RZ_LIB_PLUGINS %s\n"
+			" binrc            %s (elf, elf64, mach0, ..)\n"
+			" RZ_USER_PLUGINS  %s\n"
+			" RZ_LIB_PLUGINS   %s\n"
 			" RZ_EXTRA_PLUGINS %s\n"
 			"Environment:\n"
-			" RZ_DEBUG      if defined, show error messages and crash signal\n"
-			" RZ_DEBUG_ASSERT=1 set a breakpoint when hitting an assert\n"
-			" RZ_MAGICPATH %s\n"
-			" RZ_NOPLUGINS do not load rizin shared plugins\n"
-			" RZ_RCFILE    %s (user preferences, batch script)\n"
-			" RZ_DATAHOME %s\n"
-			" RZ_VERSION   contains the current version of rizin\n"
+			" _NT_DEBUGGER_EXTENSION_PATH  path(s) to debugger extension DLLs\n"
+			" ALACRITTY_LOG                alacritty log file's path\n"
+			" ANSICON                      ansicon's W & H of the buffer and w & h of the window in the form of: \"WxH (wxh)\"\n"
+			" CC                           compiler's name or path\n"
+			" DEBUGINFOD_URLS              e bin.dbginfo.debuginfod_urls - use alternative debuginfod server\n"
+			" COLORTERM                    extra color capabilities for a terminal\n"
+			" COLUMNS                      terminal columns to use\n"
+			" LD_LIBRARY_PATH              path(s) to search for shared libraries at run time\n"
+			" LIBRARY_PATH                 path(s) to search for static and shared libraries at compile time\n"
+			" PATH                         path(s) with executables\n"
+			" RZ_ABORTLEVEL:               target log level/severity when to abort (0:DEBUG, 1:VERBOSE, 2:INFO, 3:WARN, 4:ERROR, 5:FATAL)\n"
+			" RZ_CURL                      whether to use curl (for SSL support)\n"
+			" RZ_DEBUG                     if defined, show error messages and crash signal\n"
+			" RZ_DEBUG_ASSERT=1            set a breakpoint when hitting an assert\n"
+			" RZ_DEBUG_TOOL=gdb            debug tool to use when showing error messages and crash signal\n"
+			" RZ_DYLDCACHE_FILTER          dyld cache filter (MacOS dynamic libraries location(s) at runtime)\n"
+			" RZ_HTTP_AUTHFILE             HTTP Authentification user file\n"
+			" RZ_LOGCOLORS                 should the log output use colors\n"
+			" RZ_LOGFILE                   logging output filename/path\n"
+			" RZ_LOGLEVEL                  target log level/severity (0:DEBUG, 1:VERBOSE, 2:INFO, 3:WARN, 4:ERROR, 5:FATAL)\n"
+			" RZ_LOGSHOWSOURCES            should the log output contain src info (filename:lineno)\n"
+			" RZ_PIPE_IN                   rzpipe cmd input (file descriptor)\n"
+			" RZ_PIPE_OUT                  rzpipe cmd output (file descriptor)\n"
+			" RZ_MAGICPATH                 %s\n"
+			" RZ_NOPLUGINS                 do not load rizin shared plugins\n"
+			" RZ_RCFILE                    %s (user preferences, batch script)\n"
+			" RZ_DATAHOME                  %s\n"
+			" RZ_VERSION                   contains the current version of rizin\n"
+			" SFLIBPATH                    SFLib syscall library path\n"
+			" SHELL=sh                     shell to use (eg: \"sh\")\n"
```

Comment:
```suggestion
			" SHELL                        shell to use (eg: \"sh\")\n"
```

Reply:
Shouldn't this be applied to `RZ_DEBUG_ASSERT=1` and `RZ_DEBUG_TOOL=gdb` for consistency?